eclipse-theia/theia
# Export page objects directly under `lib`

#### What it does
Having `src` in the exported paths of the `lib` folder is sort of unexpected and weird.
Better to export it in a way so that imports for consumers look like
```typescript
import { TheiaApp } from '@theia/playwright';
```

Therefore, we define `src` as `rootDir` to get rid of the `src` in the `lib` path, add an index re-exporting all files in the root and define this index as `main` of this package.

For simplicity we also moved the tests into the `src` folder to be able to compile them consistently.

#### How to test

After a full build, you can add `@theia/playwright` as a dependency of another package in this repo and add an import to any source file of the package as follows:
```typescript
import { TheiaApp } from '@theia/playwright';
```
This import should be successfully resolved during a build.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
